src/sql/workbench/contrib/resourceViewer/browser/resourceViewerTable.ts

```diff
@@ -93,4 +102,19 @@ export class ResourceViewerTable extends Disposable {
 		}
 		return value;
 	}
+
+	private async onTableClick(event: ITableMouseEvent): Promise<void> {
+		const column = this._resourceViewerTable.columns[event.cell.cell];
+		if (column) {
+			const row = this._dataView.getItem(event.cell.row);
+			const value = row[column.field];
+			if (isHyperlinkCellValue(value)) {
+				if (isString(value.linkOrCommand)) {
```

Comment:
if openerService.open or commandService.executeCommand fails, will it show toast notification? if not, maybe we should catch the exception and show it.